packages/core/src/application.ts

```diff
@@ -192,6 +192,10 @@ export class Application extends Context {
     const instance = this.getSync<Component>(componentKey);
     mountComponent(this, instance);
   }
+
+  public packageJson(pkg: PackageJson) {
```

Comment:
I find this name confusing for users and readers.

```ts
app.packageJson(/*...*/);
```

Can we find a better name, for example `setApplicationMetadata`? (I do realize this is not consistent with other API like `app.component` or `app.controller`.)